libr/core/agraph.c

```diff
@@ -2310,12 +2310,22 @@ static void add_child(RCore *core, RAGraph *g, RANode *u, ut64 jump) {
 	}
 	char *title = get_title (jump);
 	RANode *v = r_agraph_get_node (g, title);
-	ut64 a = r_num_get (NULL, u->title);
-	ut64 b = r_num_get (NULL, title);
+	if (v) {
+		ut64 a = r_num_get (NULL, u->title);
+		int len = 64;
+		char key[len];
+		char *fmt = "agraph.edge.0x%" PFMT64x "_0x%" PFMT64x ".highlight";
+		if (len > snprintf (key, len, fmt, a, jump)) {
```

Comment:
that snprintf should never fail. you know before hand how long this string will be (2 64bit numbers in hexa are 10 chars, + a bunch of text ... so you can skip this check